src/detect_breakends/junction_detection.cpp

```diff
@@ -289,7 +289,7 @@ void detect_junctions_in_alignment_file(const std::filesystem::path & alignment_
             for (uint8_t method : methods) {
                 switch (method)
                 {
-                    case 1: // Detect junctions from CIGAR string
+                    case 0: // Detect junctions from CIGAR string
```

Comment:
You could also do the switch statement with enums:

```cpp
            for (detecting_methods method : methods) {
                switch (method)
                {
                    case detecting_methods::cigar_string: // Detect junctions from CIGAR string
```

the advantage is the verbosity and you already have the enums anyway.